code/game/verbs/ooc.dm

```diff
@@ -211,8 +211,9 @@ GLOBAL_VAR_INIT(admin_ooc_colour, "#b82e00")
 	mob.create_log(LOOC_LOG, msg)
 	if(isliving(mob))
 		for(var/mob/M in viewers(7, mob))
-			if(M.client?.prefs.toggles2 & PREFTOGGLE_2_RUNECHAT)
-				M.create_chat_message(mob, msg, FALSE, symbol = RUNECHAT_SYMBOL_LOOC)
+			if(M.client?.prefs.toggles & PREFTOGGLE_CHAT_LOOC) // Check if LOOC is enabled in game preferences
+				if((M.client?.prefs.toggles2 & (PREFTOGGLE_2_RUNECHAT|PREFTOGGLE_CHAT_LOOC)) == (PREFTOGGLE_2_RUNECHAT|PREFTOGGLE_CHAT_LOOC)) // Check if Runechat and LOOC is enabled // Check if Runechat and LOOC is enabled
```

Comment:
I apologize, I made a mistake PREFTOGGLE_2_RUNECHAT is on toggles2, while PREFTOGGLE_CHAT_LOOC is on toggles. This means these two have to be checked seperately. Basically `(M.client?.prefs.toggles2 & (PREFTOGGLE_2_RUNECHAT) && M.client.prefs.toggles & (PREFTOGGLE_CHAT_LOOC))`